bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/RepositoryImpl.java

```diff
@@ -72,6 +72,11 @@ public Node getNode() {
                 System.getProperty("os.version"));
     }
 
+    @Override
+    public boolean isRemoteEnabled() {
+        return isRemoteEnabled;
```

Comment:
Any reason why it is not covered?

Reply:
This is called from the unit test `RuntimeAPITest` and the interface method is covered. The unit test I added is failing if I remove/change the line.

Reply:
Let me analyze why the codecov fails.